lib/tree.rb

```diff
@@ -25,10 +25,6 @@ def data
       RJGit::Porcelain.ls_tree(@jrepo, @path, Constants::HEAD, options={:print => true, :io => strio})
       @contents = strio.string
     end
-    
-    def contents_array
```

Comment:
This method seems unidiomatic to me: the idiomatic way to get an `Array` for an `Enumerable` seems to be to call `.each.to_a`. So I've implemented that.

Reply:
If the object in question already is an `Enumerable`, it should already have a [to_a](https://ruby-doc.org/3.2.0/Enumerable.html#method-i-to_a) method (and the alias `entries`). Not sure what `.each.to_a` is trying to accomplish: `.each` returns an `Enumerator` which is then transformed into a kind of `Enumerable` yet again with the `to_a` call.

Reply:
You're right, I was trying to implement this behavior:

```ruby
3.0.0 :002 > [1,2].each
 => #<Enumerator: [1, 2]:each> 
```

...but of course `Enumerable` already does that for us. I did notice we were unnecessarily defining our own `#count` method (already provided by `Enumerable`), so I've removed that.